website/utils.py

```
@@ -1129,3 +1131,30 @@ def get_default_bacon_score(model_name, is_security=False):
+    When GCS is configured, this returns a signed URL valid for the
+    specified number of seconds. If GCS is not configured, a RuntimeError
+    is raised and callers should fall back to the normal media URL.
+    """
+    client = storage.Client()
+    bucket_name = getattr(
+        settings,
+        "GS_PRIVATE_BUCKET_NAME",
+        getattr(settings, "GS_BUCKET_NAME", None),
+    )
+    if not bucket_name:
+        raise RuntimeError("No GCS bucket configured for signed URL generation")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)